### PR TITLE
Add documentation for checklist resource type

### PR DESCRIPTION
**Pull request by ricardogsilva**
This PR adds some small changes to the docs to mention the new checklist resource

### CODE REVIEW

File: _posts/2016-07-22-repository-structure.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ this:
         │   ├── style (optional, containing QML files - QGIS Layer style)
         │   ├── svg (optional, containing SVG files)
         │   ├── symbol (optional, containing symbol definition XML files)
+        │   ├── checklists (optional, containing checklist definition json files)
```
Comment (havatv) [inline _posts/2016-07-22-repository-structure.md:27]:
```suggestion
        │   ├── checklists (optional, containing checklist definition JSON files)
```

File: _posts/2016-07-22-repository-structure.md [hunk 2]
```diff
@@ -36,6 +37,7 @@ this:
...
```
Comment (havatv) [inline _posts/2016-07-22-repository-structure.md:40]:
```suggestion
        │   ├── checklists (optional, containing checklist definition JSON files)
```

File: _posts/2016-07-22-repository-structure.md [hunk 3]
```diff
@@ -50,6 +52,7 @@ this:
             ├── style (optional, containing QML files - QGIS Layer style)
             ├── svg (optional, containing SVG files)
             ├── symbol (optional, containing symbol definition XML files)
+        │   ├── checklists (optional, containing checklist definition json files)
```
Comment (havatv) [inline _posts/2016-07-22-repository-structure.md:55]:
```suggestion
        │   ├── checklists (optional, containing checklist definition JSON files)
```

**Review by havatv — APPROVED** [review]
Thanks @ricardogsilva! I will merge when a new version of the plugin has been published.

**Comment by havatv** [comment]
> Thanks @ricardogsilva! I will merge when a new version of the plugin has been published.
The new version (0.16.0) has been uploaded to plugins.qgis.org, but It has not been "approved" yet.

**Comment by havatv** [comment]
I merged, since version 0.16.0 of the plugin has been approved.

**Comment by ricardogsilva** [comment]
Thanks for your work @havatv :tada:

**Comment by havatv** [comment]
> Thanks for your work @havatv tada

And thank you for the contributions, @ricardogsilva!

**Pull request merged**